app/models/concerns/twitter_user/utils.rb

```diff
@@ -0,0 +1,53 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Utils
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def has_more_than_two_records?(uid, user_id)
```

Comment:
__[rubocop]__
* Rename `has_more_than_two_records?` to `more_than_two_records?`. [:ref](https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)